repo/maintenancestats/stats_generate_range_checkpoint.go

```diff
@@ -0,0 +1,33 @@
+package maintenancestats
+
+import (
+	"fmt"
+
+	"github.com/kopia/kopia/internal/contentlog"
+)
+
+const generateRangeCheckpointStatsKind = "generateRangeCheckpointStats"
+
+// GenerateRangeCheckpointStats are the stats for generating range checkpoints.
+type GenerateRangeCheckpointStats struct {
+	FirstEpoch int `json:"firstEpoch"`
+	LastEpoch  int `json:"lastEpoch"`
```

Comment:
nit: it makes sense to name these
- `rangeMinEpoch`, `rangeMaxEpoch`; or
- `rangeStartEpoch`, `rangeEndEpoch`

```suggestion
	RangeMinEpoch int `json:"rangeMinEpoch"`
	RangeMaxEpoch int `json:"rangeMaxEpoch"`
```

Reply:
`RangeMinEpoch` and `RangeMaxEpoch` are consistent with `epoch.RangeMetadata`, thus preferred.